cmd/istioctl/main.go

```diff
@@ -679,3 +679,17 @@ func prepareClientForOthers(configs []crd.IstioKind) (*rest.RESTClient, map[stri
 	}
 	return client, resources, nil
 }
+
+func getDefaultNamespace() string {
+	configAccess := clientcmd.NewDefaultPathOptions()
+	// use specified kubeconfig file for the location of the config to create or modify
+	configAccess.GlobalFile = kubeconfig
```

Comment:
Can you pass `kubeconfig` as an argument to `getDefaultNamespace`? The less global var the better and it makes flag dependencies more obvious (see other comment on line 494).

Reply:
yes, I'll update this. good catch!